### PR TITLE
Presentation download

### PR DESCRIPTION
**Pull request by antobinary**
Included in this pull request:
* implemented download presentation from presenter and viewer's perspective (Issue #6067 #6068 )
* pulled the latest font files
* removed .eot, .svg, .ttf font types as [ [https://stackoverflow.com/questions/34452288/using-font-face-in-microsoft-edge](https://stackoverflow.com/questions/34452288/using-font-face-in-microsoft-edge) , [https://stackoverflow.com/questions/11002820/why-should-we-include-ttf-eot-woff-svg-in-a-font-face](https://stackoverflow.com/questions/11002820/why-should-we-include-ttf-eot-woff-svg-in-a-font-face) ] show we only need .woff for the device OS's and browsers we support

Notes:
* We no longer import BaseButton outside of `button/component` which means we could potentially combine the two

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/component.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  intl, handleDownloadPresentation, dark, elementName,
+}) => {
+  const formattedLabel = intl.formatMessage(
+    intlMessages.downloadPresentationButton,
+    ({ 0: elementName ? elementName.toLowerCase() : '' }),
+  );
+
+  return (
+    <div className={cx(styles.wrapper, dark ? styles.dark : styles.light)}>
+      <Button
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/component.jsx]: Why don't use a link with download attribute?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

File: bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/styles.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
+:root {
+  --color-transparent: #ff000000;
+  ::-webkit-media-controls {
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/styles.scss]: The sass compiler don't add the prefix? if not, should add to firefox `-moz-` and IE/edge`-ms-`.

File: bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/component.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  dark: false,
+  elementName: '',
+};
+
+const DownloadPresentationButton = ({
+  intl, handleDownloadPresentation, dark, elementName,
+}) => {
+  const formattedLabel = intl.formatMessage(
+    intlMessages.downloadPresentationButton,
+    ({ 0: elementName ? elementName.toLowerCase() : '' }),
```
Comment (KDSBrowne) [inline bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/component.jsx]: This isn't being used by the formatted message.

File: bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/styles.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
+:root {
+  --color-transparent: #ff000000;
```
Comment (KDSBrowne) [inline bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/styles.scss]: Could probably move this to `/imports/ui/stylesheets/variables/palette.scss`

File: bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/component.jsx (hunk 1 above)
Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/component.jsx]: We could. I saw we do that for downloading chat. However here I don't have much to pass in term of data. I don't know the mime type, I want to preserve the name, etc... Is there an issue with the Button? I do uriEncode the params

File: bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/component.jsx (hunk 3 above)
Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/component.jsx]: removed, tnx!

File: bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/styles.scss (hunk 4 above)
Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/styles.scss]: moved

File: bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/styles.scss (hunk 2 above)
Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/styles.scss]: To be honest I copied this from the fullscreen component and am not sure if it's needed and if it should be improved as you suggested. Would appreciate some input from @pedrobmarin

**Pull request merged**